static/templates/settings/data_exports_admin.hbs

```diff
@@ -29,8 +29,13 @@
         </div>
     </form>
     {{/if}}
-    <input type="hidden" class="search" placeholder="{{t 'Filter exports' }}"
-      aria-label="{{t 'Filter exports' }}"/>
+
+    <div class="settings_panel_list_header">
+        <div class="list_title">{{t "Data exports"}}</div>
+        <input type="hidden" class="search" placeholder="{{t 'Filter exports' }}"
```

Comment:
This is preexisting, but `type="hidden"` seems unlikely to be correct.

Reply:
Yeah, this is basically the only place where we hide the filter button until there are actual items in the list.  This is outside the scope of my changes, as I don't want to go down the rabbit hole of mucking with the JS here, and this is a pretty rarely used screen.  In fairness to the decision here it's quite rare to need a filter for data exports, so it's possible that the best decision is to just eliminate the filter altogether.  That's outside the scope of a code sweep.

Reply:
OK, sounds good. Eventually I think we'll want to just rework this to be a formal component that works more like `settings_checkbox`, where you just pass in a couple options, and features like that hiding detail can be part of the common system. Agree it isn't something we should change here, as long as it's not obviously a bug.